exampleCourse/questions/template/multiple-choice/all-of-the-above/question.html

```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+  <p>Which of the following are considered fundamental programming concepts?</p>
+</pl-question-panel>
+
+<pl-multiple-choice answers-name="cs-concepts">
+  <pl-answer correct="false">Variables</pl-answer>
+  <pl-answer correct="false">Loops</pl-answer>
+  <pl-answer correct="false">Conditionals</pl-answer>
+  <pl-answer correct="true">All of the above</pl-answer>
```

Comment:
Same as the other PR, you should use the all of the above option on the element instead of explicitly adding this answer choice.